nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/processors/AbstractJMSProcessor.java

```diff
@@ -118,6 +118,23 @@
             .defaultValue(Charset.defaultCharset().name())
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
+    static final PropertyDescriptor ALLOW_ILLEGAL_HEADER_CHARS = new PropertyDescriptor.Builder()
+            .name("Allow Illegal Characters in Header Names")
```

Comment:
can you use ``.displayName()`` and ``.name()`` with a programmatic friendly name for the "name"?